react/features/app/actions.js

```diff
@@ -49,6 +47,15 @@ export function appNavigate(uri) {
                     .domain;
         }
 
+        if (room) {
```

Comment:
Because we can pass urls like `meet.jit.si/testRoom` and constructor of URL class will create URL object for address like 'https://meet.jit.si/meet.jit.si/testRoom'. Splitting and getting the last element guarantees that we're taking only pathname + search + hash but not the domain. WDYT?